lib/libbackscrub.h

```diff
@@ -0,0 +1,54 @@
+/* This is licenced software, @see LICENSE file.
+ * Authors - @see AUTHORS file.
+==============================================================================*/
+
+// tested against tensorflow lite v2.4.1 (static library)
+#ifndef _LIBBACKSCRUB_H
+#define _LIBBACKSCRUB_H
+
+// for cv::Mat and related types
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
```

Comment:
Maybe avoid those here in the public interface to allow for a plain C interface.

Reply:
Yep, as above C-compatible API possible once more state is hidden, although we are presently passing `cv::Mat` objects for raw video frames, and may need to use the plain C alternative from OpenCV...

Reply:
As above, no C compatible replacement exists for `cv::Mat` objects:
https://stackoverflow.com/questions/48365991/is-there-a-working-c-interface-for-opencv-3-x

Reply:
okay, will have to refactor this part later on (got some idea on how to get this down to plain old C). We just should keep this in mind for now to reduce the work on this subject later on.